src/index.js

```diff
@@ -206,7 +207,39 @@ function sanitizeHtml(html, options, _recursing) {
                 return;
               }
             }
-
+            if (name === 'iframe' && a === 'src') {
+              try {
+                var parsed = url.parse(value)
+                var whitelistedDomains = options.allowedIframeDomains.find(function(domain) {
+                  return domain === parsed.hostname;
+                })
+                if (!whitelistedDomains) {
+                  //check the href to see if the hostname didn't get picked up, doesn't mean it's invalid
```

Comment:
This seems to be about the issue with URLs that start with //. If so you should be fixing the URL by adding the https: before passing it to url.parse, less hacky that way.

Reply:
Yeah i was trying to think of other ways the url might be presented but it seems that this is the one issue. Now I check for `http:` and `https:` and just `concat` the value of it if it doesn't exist and starts with `//`